hedyorg/hedy
# 💻 Add checkbox in view programs page

Adds a checkbox in the view programs page, only visible if the user is the teacher of the creator of the program. By ticking the checkbox, the program is stored in the database as reviewed. By going backwards to the table, it'll trigger a reload and the page will be updated. At last, I also added a tick mark in the adventure page, that lets the student know if the teacher reviewed this program.

Fixes #5752 

**How to test**

1. Log in as a student and run a program in several tabs (can't be the examples, because otherwise it won't be shown in the table)
2. Now log in as the teacher and go to the class page where the student belongs. Click the eye icon and then tick the checkbox.
3. The checkbox for this program should be ticked.
